src/fixtures/demos.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { Navigate } from 'react-router-dom'
 
+import Demo1 from '../../src/components/Demo1/Index'
```

Comment:
Again, please rename this to index so in this import you can use

`import Demo1 from '../../src/components/Demo1'`